tests/two_d_line_tests.cpp

```diff
@@ -106,7 +106,7 @@ static std::string ColorFormatName(uint32_t format, bool ReturnShortName) {
       if (ReturnShortName) return "15";
       return "R5G5B5";
     case NV05C_SET_COLOR_FORMAT_LE_X8R8G8B8:
-      if (ReturnShortName) return "32";
+      if (ReturnShortName) return "24";
```

Comment:
Super nit (ignore it for this PR) - this change doesn't really have anything to do with the other fixes, so ideally it'd be a separate PR.

Reply:
next time!